crates/networking/manager/src/gossipsub/handle_gossipsub.rs

```diff
@@ -78,8 +93,27 @@ pub async fn handle_gossipsub_message(message: Message, beacon_chain: &BeaconCha
                     signed_block.message.block_root()
                 );
 
-                if let Err(err) = beacon_chain.process_block(*signed_block).await {
-                    error!("Failed to process gossipsub beacon block: {err}");
+                let Ok(validation_result) =
+                    validate_beacon_block(beacon_chain, cached_db, &signed_block).await
+                else {
+                    return;
```

Comment:
Does this simply ignore `anyhow::Error` from `validate_beacon_block()` e.g. below?

```rs
    let validator = latest_state_in_db
        .validators
        .get(block.message.proposer_index as usize)
        .ok_or(anyhow!("Invalid proposer index"))?;
```

```rs
    if !latest_state_in_db.verify_block_signature(block)? {
        return Ok(ValidationResult::Reject);
    }
```

```rs